manifest_staging/charts/aad-pod-identity/templates/nmi-daemonset.yaml

```diff
@@ -138,15 +138,10 @@ spec:
             name: metrics
             protocol: TCP
         {{- end }}
+        {{- with .Values.nmi.containerSecurityContext }}
         securityContext:
-          runAsUser: 0
```

Comment:
We would need `runAsUser: 0` no matter what or else both mic and nmi won't work. Can we not put `runAsUser: 0` as part of values.yaml, instead, let's keep it here and give users ability customize other security-context-related settings. What do you think?

Reply:
Yeah, sure, I will change that.